Fixes #47 (#229)
knatten/cppquiz23: questions/220/explanation.md

`f()` evaluates to false, so the value of the whole conditional expression is the value of the third expression `h()`, that is the character `h`. But which functions are evaluated, and in which order?

Let's have a look at §[expr.cond]¶1 in the standard:

> > Every value computation and side effect associated with the first expression is sequenced before every value computation and side effect associated with the second or third expression.

Reviewer: ```suggestion
> The first expression is sequenced before the second or third expression (§[intro.execution]).
```

Author: Oops, thanks!
